numba_cuda/numba/cuda/tests/core/test_serialize.py

```diff
@@ -60,13 +59,15 @@ def check_result(func):
             new_func = pickle.loads(pickled)
             check_result(new_func)
 
+    @skip_on_standalone_numba_cuda
```

Comment:
If you have the skip decorator on a class you don't need it on the individual test cases as well (applies to all tests in this class).